Crashlytics/Crashlytics/Settings/FIRCLSSettingsOnboardingManager.m

```diff
@@ -199,32 +150,6 @@ - (void)operation:(FIRCLSDownloadAndSaveSettingsOperation *)operation
   NSTimeInterval currentTimestamp = [NSDate timeIntervalSinceReferenceDate];
   [self.settings cacheSettingsWithGoogleAppID:self.googleAppID currentTimestamp:currentTimestamp];
 
-  // only try this once
-  if (self.hasAttemptedAppConfigure) {
-    FIRCLSDebugLog(@"App already onboarded in this run of the app");
-    [self finishNetworkingSession];
-    return;
-  }
-
-  // Onboarding is still needed in Firebase, here are the backend app states -
-  // 1. When the app is created in the Firebase console, app state: built (client settings call
-  // returns app status: new)
-  // 2. After onboarding call is made, app state: build_configured
-  // 3. Another settings call is triggered after onboarding, app state: activated
-  if ([self.settings appNeedsOnboarding]) {
```

Comment:
I believe these are the only places these settings are being used. We won't use them in the future (hopefully). Can we remove them from the settings model entirely so we don't confuse ourselves later (can remove all 3 of these methods I believe): https://github.com/firebase/firebase-ios-sdk/blob/master/Crashlytics/Crashlytics/Models/FIRCLSSettings.m#L264-L276